src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/OppFinder/WhenHandlingGetOppFinderApprovedStandardDetailsRequest.cs

```diff
@@ -117,33 +134,60 @@ public async Task ThenReturnsRegionDetails()
             var result = await _sut.Handle(request, new CancellationToken());
 
             //Assert
-            result.RegionResults.Count.Should().Be(2);
+            result.VersionResults.Count.Should().Be(2);
             using (new AssertionScope())
             {
-                result.RegionResults[0].Region.Should().Be("REGION1");
-                result.RegionResults[1].Region.Should().Be("REGION2");
+                result.VersionResults[0].Version.Should().Be("1.0");
+                result.VersionResults[1].Version.Should().Be("1.1");
             }
         }
 
         [Test]
-        public async Task ThenReturnsVersionDetails()
+        [TestCase("eqaProviderContactEmail@email.com", "eqaProviderContactEmail@email.com", "", "OFQUAL", "eqaProviderContactEmail@email.com")]
+        [TestCase("eqaProviderContactName@email.com", "", "eqaProviderContactName@email.com", "OFQUAL", "eqaProviderContactName@email.com")]
+        [TestCase("", "", "", "OFQUAL", "OFQUAL")]
+        [TestCase("eqaProviderContactName", "", "eqaProviderContactName", "OFQUAL", "OFQUAL")]
+        [TestCase("eqaProviderContactEmail", "eqaProviderContactEmail", "", "OFQUAL", "OFQUAL")]
+        public async Task ThenReturnsEqaProvider(string validationRequestEmail, string eqaProviderContactEmail, string eqaProviderContactName, string eqaProviderName, string eqaProvider)
         {
             //Arrange
             var request = new GetOppFinderApprovedStandardDetailsRequest()
             {
                 StandardReference = "ST0001"
             };
 
+            var validationRequest = new ValidationRequest()
+            {
+                Type = "email",
+                Value = validationRequestEmail
+            };
+
+            ValidationHandler validationHandler = new ValidationHandler(new ValidationService());
```

Comment:
You should be mocking the validation handler instead of using the real one so mock for "" returns true, "valid@email.com" returns true and "invalid" returns false, then use those values for the inputs, otherwise you are dependent on the implementation of the validation handler in this test

Reply:
Removed validation handler and instead mocked with valid values being "", null, "valid@email.com" and invalid being "invalid"